src/st/sel/save_mgr2.c

```diff
@@ -1,20 +1,21 @@
 #include "sel.h"
+#include "memcard.h"
 
 void _clear_event(void);
 void _clear_event_x(void);
 s32 _peek_event_with_retry(void);
 
-s32 func_801B8A8C(s32 arg0, s32 arg1) {
+s32 func_801B8A8C(s32 slot, s32 arg1) {
     s32 temp_s2;
-    Unkstruct_801B8A8C* unk;
-    temp_s2 = (arg0 * 0x10) + arg1;
-    unk = &D_801BC3F0[arg0];
-    switch (D_801D6B20) {
+    MemcardInfo* memcard;
+    temp_s2 = (slot * 0x10) + arg1;
```

Comment:
Seems like `arg1` should be `cardNumber`?
`_card_info(long chan): Port number x 16 + Card number`

Maybe `chan = (slot * 0x10) + cardNumber;`

cardNumber maybe nonzero when a multitap is connected

Reply:
I was just going off this 
<img width="784" alt="image" src="https://github.com/Xeeynamo/sotn-decomp/assets/122322823/6aeab6d7-71e5-4988-a9fb-2eb4599b5512">
I suppose I'd be inclined to go with the terminology from the manual: port_num and card_num or whatever, but I think the current ones are fine